plugins/outputs/azure_monitor_logs/azure_monitor_logs_test.go

```diff
@@ -0,0 +1,277 @@
+package azure_monitor_logs
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/stretchr/testify/require"
+)
+
+func getMetric(name string) telegraf.Metric {
+	m, err := metric.New(
+		name,
+		map[string]string{
+			"host": "test-machine",
+			"env":  "testing",
+		},
+		map[string]interface{}{
+			"value": 42.0,
+		},
+		time.Unix(0, 0),
+	)
+	if err != nil {
+		panic(err)
+	}
+	return m
+}
```

Comment:
This is equivalent to 
```suggestion
testutil.MustMetric(
	name,
	map[string]string{
		"host": "test-machine",
		"env":  "testing",
	},
	map[string]interface{}{"value": 42.0},
	time.Unix(0, 0),
)
```